txbits/app/models/UserModel.scala

```diff
@@ -57,7 +57,7 @@ class UserModel(val db: String = "default") {
   }
 
   def addFakeMoney(uid: Long, currency: String, amount: BigDecimal) = DB.withConnection(db) { implicit c =>
-    if (Play.current.configuration.getBoolean("fakeexchange").get) {
+    if (globals.fakeExchange) {
```

Comment:
The test changes the configuration and starts a fake application.

On February 1, 2015 11:01:55 AM EST, Kirk Zathey <notifications@github.com> wrote:
>> @@ -57,7 +57,7 @@ class UserModel(val db: String = "default") {
>>    }
>>  
>>    def addFakeMoney(uid: Long, currency: String, amount: BigDecimal)
>= DB.withConnection(db) { implicit c =>
>> -    if (Play.current.configuration.getBoolean("fakeexchange").get) {
>> +    if (globals.fakeExchange) {
>
>Moving this to a variable in Global.scala makes the unit test `not be
>able to add fake balance on real exchange` fail. Any idea why?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/txbits/txbits/pull/68/files#r23898550

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.